Resources/Locale/en-US/_NF/nutrition/components/food-component.ftl

```diff
@@ -0,0 +1,2 @@
+food-system-nasty = This food gross.
+food-system-toxin = This food bad.
```

Comment:
Grammatical nitpicks.  These show up after eating food, yes?

```suggestion
food-system-nasty = That food was gross.
food-system-toxin = That food was bad.
```

It might be interesting to have multiple lines of text to avoid repetition.  Here are some stronger descriptors per type.

Nasty:
That food was disgusting.
That food was repulsive.
That tasted vile.

Toxin:
Your stomach starts to churn.
You feel nauseous after eating that.
You can barely keep that food down.